tempesta_fw/cache.c

```diff
@@ -1579,6 +1579,9 @@ tfw_cache_build_resp(TfwHttpReq *req, TfwCacheEntry *ce)
 	if (tfw_cache_build_resp_body(db, resp, trec, &it, p))
 		goto err;
 
+	if (TFW_CONN_TLS(req->conn))
```

Comment:
This should happen right after the response object is allocated in `tfw_cache_build_resp()` function, i.e. after `tfw_http_msg_setup()`.  Please add a comment here, why the flag is added and which problem it solves.

And we have problem here, only very  first skb is marked with the flag, all the following skbs in the message won't have the flag. Bit in the same time huge responses may require more than one skb. Seems like we should care of this case in `tfw_msg_iter_append_skb()` function.

Reply:
DONE